### ISSUE TITLE
False positive at multiline-if-else when using "else if"

### ISSUE THREAD

**Issue by carstenhag** [pinterest/ktlint#1904]
## Expected Behavior
```
else if () {
}
```
Should stay as is

## Observed Behavior
It gets formatted to
```
else {
   if () {
   }
}
```

## Steps to Reproduce
```
    val x = if (showFilter) {
        1
    } else if (activeTabIndex.value == 0) {
        2
    } else {
        null
    } ?: 0
```

-->

```
    val x = if (showFilter) {
        1
    } else {
        if (activeTabIndex.value == 0) {
            2
        } else {
            null
        } ?: 0
    }
```

## Your Environment
* Version of ktlint used: 0.48.2
* Relevant parts of the `.editorconfig` settings
* Operating System and version: macOS

**Comment by paul-dingemans** [comment]
Tnx for reporting. It is just in time to be fixed in next release.

**Comment by paul-dingemans** [comment]
Similarly the example below should be accepted without reformatting as well:
```
val foo = if (bar1) {
    "bar1"
} else if (bar2) {
    "bar2"
} else {
    "bar3"
}.plus("foo")
```
